src/views/SelectProduct.vue

```diff
@@ -2,79 +2,43 @@
   <ion-page>
     <ion-header :translucent="true">
       <ion-toolbar>
-        <ion-menu-button slot="start" />
+        <ion-menu-button slot="start" id="first" menu="first" />
```

Comment:
name should be relevant and representative of the content inside the menu they open. Something like "Navigation"

Reply:
I have renamed menu id of first menu to navigation and last menu to filters